ui/components/app/flask/experimental-area/index.scss

```
@@ -0,0 +1,44 @@
+.experimental-area {
+  color: $flask-purple;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .logo {
+    padding: 16px;
+    line-height: 0.625rem;
+    font-family: monospace;
+  }
+
+  .experimental-text {
+    padding: 16px;
+    font-family: monospace;
+    margin: auto;
+    line-height: 16px;
+  }
+
+  .text {
+    padding: 16px;
+    max-width: 670px;
+  }
+
+  ul {
+    padding: 16px 0;
+  }
+
+  li {
+    list-style-type: disc;
+    padding-left: 8px;
+    list-style-position: inside;
```

Adds a little more space between the list items
```suggestion
    list-style-position: inside;
    margin-bottom: 8px;

    &:nth-last-child(1) {
      margin-bottom: 0;
    }
```